app/views/admins/users/edit.html.slim

```diff
@@ -1,13 +1,68 @@
-= form_for @user do |f|
-  .field
-    = f.label :email_col
-    br
-    = f.text_field :email_col
-  .actions
-    = f.submit
+= form_for @user, url: { action: "update"} do |f|
+  - invalid = @user.errors.include?(:email)
```

Comment:
please consider [THIS APPROACH](https://apidock.com/rails/ActionView/Helpers/ActiveRecordHelper/error_messages_for)